teambit.bvm/commands/version/version.ts

```diff
@@ -128,22 +146,33 @@ function formatOutput(versions: VersionsResult): string {
 function getNewerBitAvailableOutput(
   currentVersion?: string,
   latestInstalledVersion?: string,
-  latestRemoteVersion?: string
+  latestRemoteStableVersion?: Version,
+  latestRemoteNightlyVersion?: Version
 ): string | undefined {
   if (!currentVersion) {
     return undefined;
   }
-  if (!latestInstalledVersion && !latestRemoteVersion) {
+  if (!latestInstalledVersion && !latestRemoteStableVersion && !latestRemoteNightlyVersion) {
     return undefined;
   }
-  if (
-    semver.gt(latestInstalledVersion, currentVersion) ||
-    (latestRemoteVersion && semver.gt(latestRemoteVersion, currentVersion))
-  ) {
-    return `new version of ${chalk.cyan(
-      "bit"
-    )} is available, upgrade your ${chalk.cyan("bit")} by running "${chalk.cyan(
-      "bvm upgrade"
-    )}"`;
+  function newVersionAvailableText(versionToCheck?: Version){
+    if (!versionToCheck) return undefined;
+
+    const commandToRun = versionToCheck.releaseType === ReleaseType.STABLE ?
+      `bvm install ${versionToCheck.version}`
+      : "bvm upgrade"
+    
+    if (
+      semver.gt(latestInstalledVersion || "0.0.0", currentVersion || "0.0.0") ||
+      (versionToCheck.version && semver.gt(versionToCheck.version, currentVersion || "0.0.0"))
+    )
+    {
+      return `new ${versionToCheck.releaseType?.toString() ?? ""} version (${versionToCheck.version}) of ${chalk.cyan("bit")} `+
+           `is available, upgrade your ${chalk.cyan("bit")} by running "${chalk.cyan(commandToRun)}"\n`;
+    }
   }
+  latestRemoteStableVersion = undefined;
+  const output = [newVersionAvailableText(latestRemoteStableVersion), newVersionAvailableText(latestRemoteNightlyVersion)].join("");
+    
+  return output;
```

Comment:
we should probably write some snapshot tests for these output generating functions.

Reply:
They're just the existing code turned into a function to save on re-use. Isn't a snapshot test for that a bit overkill?

We should probably have a unittest for the whole functionality but bvm isn't well tested (inc this file) and it's probably not in the scope of this task to remedy that...